### ISSUE TITLE
[Web] [Accessibility][A11yMAS][Vision Impairment][MAS4.3.1][Social media icons are not visible in High contrast white mode]

### ISSUE THREAD

**Issue by DasaradhaKamjula** [OfficeDev/script-lab#690]
**User Experience:**
User with vision impairment will not be able to see if controls are not visible in High contrast mode.

**Test Environment:**
OS Build: 16299.19
Browser: Microsoft Edge 41.16299.15.0
Tools Used: High contrast White
**Attachments:**
![mas4 3 1_social media icons](https://user-images.githubusercontent.com/33054846/32430546-c612b962-c2f5-11e7-8a0f-44a9247d6707.png)

**Repro Steps:**
1. Open the URL: http://scriptlabdocsdemo.azurewebsites.net/Office%20Dev%20Center%20-%20Docs%20-%20Build%20your%20first%20Excel%20add-in.html .
2.  Turn on High contrast settings and Navigate to social media icons.
3. Verify whether Social media icons are  visible in High contrast white mode or not. 
 
**Current Behavior:** 
Social media icons are not visible in High contrast white mode.

**Expected Behavior:**
Social media icons should be visible in High contrast white mode, so that user will be able to see all controls.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={672c8a8b-ed4f-463f-868f-e8131caf9b74}

**Suggested Fix:**
https://microsoft.sharepoint.com/teams/msenable/mas/pages/mas-details.aspx?masid=4.3.1

**Comment by Hongbo-Miao** [comment]
Why this one is posted here? It is not related with Script Lab

**Comment by Zlatkovsky** [comment]
I spoke with Toby about this (and in fact, I see she added "external" as a label to this).
There was a misunderstanding between what is an accessbility issue related to Script Lab, and what isn't.  Esp. since the page itself is a mock with incomplete JS/CSS references, so it might not even be a real bug -- just an artifact of the way I currently have coded it up for the demo.

Closing the issue.  @Tales-of-T , or @DasaradhaKamjula , feel free to close other such issues that relate to the outer chrome of the demo page rather than to the iframe (for that particular demo page).
